kjourney/src/app/main/main.component.ts

```diff
@@ -11,13 +13,17 @@ import { Role } from '../../models/role'
 export class MainComponent implements OnInit {
   private data: DataService
   public roles: Role[]
+  public capabilities: Capability[]
+  public jobFamilies: JobFamily[]
 
   constructor(data: DataService) {
     this.data = data
   }
 
   async ngOnInit() {
     this.roles = await this.data.getRoles()
+    this.capabilities = await this.data.getCapabilities()
```

Comment:
change this to subscribe to the observables from the DataService instead

Reply:
I second this